extensions/processing-parameters/README.md

```diff
@@ -0,0 +1,56 @@
+# Processing Parameters Extension
+
+The Processing Parameters Extension to the openEO API provides an interface to explore and handle additional processing options that a back-end can offer for the three processing modes (synchronous processing, batch jobs, secondary web services).
+
+- Version: **0.1.0**
+- Stability: **experimental**
+- [OpenAPI document](openapi.yaml)
+- Conformance class: `https://api.openeo.org/extensions/processing-parameters/0.1.0`
+
+**Note:** This document only documents the additions to the specification.
+Extensions can not change or break existing behavior of the openEO API.
+
+This extension adds a new endpoint (`GET /processing_options`, see [OpenAPI document](openapi.yaml))
+to discover the additional processing options that a back-end offers.
+
+Additionally, this extension allows to provide specific default values for user-defined processes (UDPs, see below).
+
+In both cases, the parameters and its values are provided separately for each processing mode.
+
+## Processing Parameters in UDPs
+
+UDPs can provide default values for specific processing parameters.
+
+The values for each parameter are provided  separately for each processing mode.
+The following properties are added to the top-level of a UDP for the respective processing modes:
+
+- `default_synchronous_parameters` for synchronous processing
+- `default_job_parameters` for batch jobs
+- `default_service_parameters` for secondary web services
+
+The schema for each of these properties is:
+
+```yaml
+type: object
+additionalProperties:
+  description: Any type
+```
+
+The keys of the object are the respective parameter names.
+The values of the object are the default values for the parameters.
+Schematic restrictions are not defined for the object, but the schemas for the parameters as defined in `GET /processing_options` apply to the given values.
+These values provide the defaults unless a user overrides them in the actual data processing request (e.g. `POST /jobs`, see below).
+
```

Comment:
Maybe add an example here to make things a bit more concrete:

```
Example process definition including processing option defaults:

{
    "id": "my_evi",
    "parameters": [...],
    "process_graph": {...},
    "default_job_parameters": {
        "memory": "2GB",
        "block-sizes": [128, 32]
    }
}
```

Reply:
See https://github.com/Open-EO/openeo-api/pull/471/commits/a407db56dcb22633f218622d547f7c97dc61f057